api/src/convert/tx_blueprint_output.rs

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) 2018-2025 The MobileCoin Foundation
+
+//! Convert to/from mc_transaction_builder::TxBlueprintOutput.
+
+use crate::{external, ConversionError};
+use mc_transaction_builder::TxBlueprintOutput; // Import only the main enum
+use mc_transaction_core::encrypted_fog_hint::EncryptedFogHint;
+// Removed Amount, UnmaskedAmount, tx::TxOut as unused
```

Comment:
not sure if these were early developer notes or something else, but they feel out of place

```suggestion
use mc_transaction_builder::TxBlueprintOutput;
use mc_transaction_core::encrypted_fog_hint::EncryptedFogHint;
```